assets/js/router.js

```diff
@@ -0,0 +1,18 @@
+var AppRouter = Backbone.Router.extend({
+  initialize: function() {
+    this.collection = new Blogs();
+
+  },
+
+  routes: {
+    '': 'index',
+    ':id/view': 'view',
+  },
+
+  view: function(id) {
```

Comment:
Your view function should look up a single model using `Backbone.Collection.prototype.get` and passing in the `id`